src/user/user.service.ts

```diff
@@ -25,29 +29,48 @@ export class UserService {
     return createdUser.save();
   }
 
+  async registerUser(userData: CreateUserDto) {
+    try {
+      if (
+        !userData.email ||
+        !userData.password ||
+        !userData.name ||
+        !userData.lastname ||
+        !userData.phone
+      ) {
+        throw new HttpException(
+          MISSING_REQUIRED_FIELDS,
+          HttpStatus.UNPROCESSABLE_ENTITY,
+        );
+      }
```

Comment:
Why perform these additional checks if the controller already has a DTO that verifies the necessary data?